applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/mechanical_explicit_strategy.hpp

```diff
@@ -461,6 +486,103 @@ class MechanicalExplicitStrategy
     ///@name Private Operations
     ///@{
 
+    /**
+     * @brief This method computes the explicit contribution of the constraints
+     * @param pScheme The pointer to the integration scheme used
+     * @param rModelPart The model part which defines the problem
+     */
+    void ComputeExplicitConstraintConstribution(
+        typename TSchemeType::Pointer pScheme,
+        ModelPart& rModelPart
+        )
+    {
+        // The current process info
+        ProcessInfo& r_current_process_info = rModelPart.GetProcessInfo();
```

Comment:
const, the Constraints already only accept const ProcessInfo (unlike Elements and Conditions which need #2993 for this)